app/assets/stylesheets/omni_nav/omni_nav_v2/omni-nav.scss

```diff
@@ -10,32 +10,18 @@ html.omni-nav-v2 {
     padding-top: 120px;
   }
 
-  @include media($large-screen) {
+  @include media($xl-screen) {
     padding-top: 60px;
   }
 
-  /* branded omninav always has primary nav stacked on top of global nav */
```

Comment:
Unbranded and branded pages used to have different breakpoints but now they share one, which is why there are quite a few deletions in this area